### DISCUSSION TITLE
Brazilian Portuguese locale and translation + other future translations

### DISCUSSION THREAD

**Discussion by LuwkasLima** [family/connectkit#79] [Show and tell]
Hi Folks,

First of all, would like to congratulate you all for the great work here. I've been looking for a wallet kit for one of the DApps I'm developing to be used in Latin America and again the language is a barrier. You have added the bare minimum for the open source community to contribute with translations upfront and that's what I call true inclusion.

Going straight to the topic, I've cloned the repo, created a pt-BR translation and updated all required fields. To be respectful of the guidelines, I'm posting it here first before I send a PR for review. Bellow some important details:

- I've worked on the pt-BR translation for the moment, but I also plan to contribute with the pt-PT and reviewing es-ES. Once PR is good and I'm aligned with the process I'll jump to the other ones. (I speak all these languages btw)
- I've noticed one of the links provided in all the locales isn't valid anymore, so I'll also update it on the next PR.
- For very specific terms related to Ethereum, I've used the pt-br version of Ethereum’s documentation, so the term is in alignment with Ethereum docs.

That's all I have. I'm willing to participate more if the community here is open for it. It's been years that I don't contribute to open source communities and please consider me as a newbie.

**Comment by lochie** [comment]
Hey @LuwkasLima! Thanks for being so enthusiastic about ConnectKit, we're glad you like it! 😀

You're welcome to submit a PR with the `pt-BR` translations. We would very much appreciate the contribution ❤️

If you have any questions about how to implement the translations feel free to raise them in this thread.

Thanks again!
